src/index.ts

```diff
@@ -74,13 +87,13 @@ export const estimateAbility = (
   function estimateAbilityEAP() {
     let num = 0;
     let nf = 0;
-    for (let i = 0; i < prior.length; i++) {
-      const theta = prior[i][0];
-      const probability = prior[i][1];
-      const like = likelihood(theta);
+    let theta, probability, like;
+    prior.forEach((e) => {
+      theta = e[0];
+      probability = e[1];
+      like = likelihood(theta);
```

Comment:
```suggestion
    prior.forEach(([theta, probability]) => {
      const like = likelihood(theta);
```

I'd suggest moving the declarations for theta, probability, and like inside of the `forEach` function to limit their scope.